index.html

```diff
@@ -0,0 +1,186 @@
+         taking place in October, in Rwanda.
+            </p>
+            <div class="date">2023.10.15(THU) ~ 16(FRI)</div>
+            <div class="location">
+              @ Kigali Convention Center
+            </div>
+          </section>
+    </div>
+    
+    <!-- Main program section -->
+    <section class="section-2" id="program">
+        <div class="h2-red-line">
+            <h2 class="h2">Main Program</h2>
+            <hr class="red-line">
+        </div>
+
+        <div class="container">
+            <!-- Box 1 -->
+            <div class="inner-container">
+                <img src="images/program_icon_01.png" alt="image" class="icon">
+                <h3 class="h3">Lecture</h3>
+                <p class="p">
+                    Listen to the speakers from countries about the messages of sharing
+                    and opening.
+                  </p>
+              </div>
+
+                  <!-- Box 2 -->
+                  <div class="inner-container">
+                    <img class="icon" src="images/program_icon_02.png" alt="image" />
+                    <h3 class="h3">TMS Exibition</h3>
+                    <p class="p">
+                      Appreciate various creations applying TMS license of artists,
+                      organized from Art Center Nabi.
+                    </p>
+                  </div>
+        
+                  <!-- Box 3 -->
+                  <div class="inner-container">
+                    <img class="icon" src="images/program_icon_03.png" alt="image" />
+                    <h3 class="h3">Forum</h3>
+                    <p class="p">
+                      Have the time to share your thoughts and opinions with experts for
+                      each topic.
+                    </p>
+                  </div>
+        
+                  <!-- Box 4 -->
+                  <div class="inner-container">
+                    <img class="icon" src="images/program_icon_04.png" alt="image" />
+                    <h3 class="h3">Hands-On Workshops:</h3>
+                    <p class="p">
+                      Try creating your own work using open source license rather than
+                      just watching at it.
+                    </p>
+                  </div>
+        
+                  <!-- Box 5 -->
+                  <div class="inner-container">
+                    <img class="icon" src="images/program_icon_05.png" alt="image" />
+                    <h3 class="h3">Networking Opportunities</h3>
+                    <p class="p">
+                      Get opportunities to network with TMS affiliates around the world,
+                      also after the summit.
+                    </p>
+                  </div>
+                </div>
+        
+                <div class="red-box">Join  TechMingle Summit 2023</div>
+                <div class="whole-program">SEE THE WHOLE PROGRAM</div>
+              </section>
```

Comment:
- [ ] Good job trying to fix the change mentioned by the previous reviewer. However, while you were fixing it, I noticed that you mistakenly damage the original design on the `DESKTOP`. I suggest you all of them and re-check before submitting to make sure all are correct.
### Template Design
![Screenshot from 2023-08-02 18-03-43](https://github.com/gilberthappi/TechMingle_Summit/assets/83042742/af85c4d7-e05b-43a2-93ff-5528c7490fb5)

### Your Design
![Screenshot from 2023-08-02 17-51-17](https://github.com/gilberthappi/TechMingle_Summit/assets/83042742/342d3656-f4a3-492e-827b-3d93e8a823be)

Reply:
Okey Thanks